### PR TITLE
Bill information

### PR DESCRIPTION
**Pull request by ra3738**
Displays all bill information for a user

### CODE REVIEW

File: client/src/components/AuthenticatedApp.jsx [hunk 1]
```diff
@@ -81,8 +82,15 @@ const AuthenticatedApp = () => {
                    () => (
                      <DoctorViewer />
                    )
-                 }
-              />
+                 }/>
+              <Route
+                 path='/users'
```
Comment (RachitMalik12) [inline client/src/components/AuthenticatedApp.jsx:87]: maybe change this to another path

File: client/src/components/AuthenticatedWelcomePage.jsx [hunk 2]
```diff
@@ -26,48 +26,55 @@ const AuthenticatedWelcomePage = () => {
...
+      <Grid item xs={3}>
         <Card className={classes.root}>
           <CardActionArea onClick={() => history.push('/clinics')}>
             <CardMedia
               className={classes.media}
               image='https://blogs.ubc.ca/theedventure/files/2015/12/de1_0506.jpg'
             />
             <CardContent>
-              <Typography gutterBottom variant='h6' component='h2'>
+              <Typography gutterBottom variant="h6" component="h2" align="center">
```
Comment (RachitMalik12) [inline client/src/components/AuthenticatedWelcomePage.jsx:39]: single quotes pls: minor. Maybe this is eslint.

File: client/src/components/AuthenticatedWelcomePage.jsx [hunk 3]
```diff
@@ -26,48 +26,55 @@ const AuthenticatedWelcomePage = () => {
...
-    </Grid>
-  );
-};
-
+      <Grid item xs={3}>
+        <Card className={classes.root}>
+          <CardActionArea onClick={()=> history.push('/users')}>
+          <CardContent>
+              <Typography gutterBottom variant="h1" component="h2" align="center">
+                $
```
Comment (RachitMalik12) [inline client/src/components/AuthenticatedWelcomePage.jsx:66]: Maybe you can use a $ or money emoji: https://emojipedia.org/dollar-banknote/

Comment (akhtariev) [reply client/src/components/AuthenticatedWelcomePage.jsx:66]: you like emoji huh?

**Pull request merged**